e2e/portalicious/pages/PaymentsPage.ts

```diff
@@ -0,0 +1,106 @@
+import { expect } from '@playwright/test';
+import { Locator, Page } from 'playwright';
+
+import TableComponent from '@121-e2e/portalicious/components/TableComponent';
+import BasePage from '@121-e2e/portalicious/pages/BasePage';
+
+class PaymentsPage extends BasePage {
+  readonly page: Page;
+  readonly table: TableComponent;
+  readonly createPaymentButton: Locator;
+  readonly addToPaymentButton: Locator;
+  readonly fspSummary: Locator;
+  readonly startPaymentButton: Locator;
+  readonly paymentTitle: Locator;
+  readonly paymentSummaryMetrics: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.page = page;
+    this.table = new TableComponent(page);
+    this.createPaymentButton = this.page.getByRole('button', { name: 'Pay' });
+    this.addToPaymentButton = this.page.getByRole('button', {
+      name: 'Add to payment',
+    });
+    this.startPaymentButton = this.page.getByRole('button', { name: 'Start' });
+    this.fspSummary = this.page
+      .locator('p-card')
+      .filter({ hasText: 'Financial Service Provider(s' });
+    this.paymentTitle = this.page.getByRole('link', { name: 'Payment' });
+    this.paymentSummaryMetrics = this.page
+      .getByTestId('payment-summary-metrics')
+      .locator('app-metric-container');
+  }
+
+  async selectAllRegistrations() {
+    await this.table.selectAllCheckbox();
+  }
+
+  async validatePaymentSummary({
+    fsp,
+    registrationsNumber,
+    paymentAmount,
+  }: {
+    fsp: string[];
+    registrationsNumber: number;
+    paymentAmount: number;
+  }) {
+    const fspSummary = await this.fspSummary.textContent();
+
+    expect(fspSummary).toContain(
+      `Financial Service Provider(s): ${fsp.join(', ')}`,
+    );
+    expect(fspSummary).toContain(`Registrations: ${registrationsNumber}`);
+    expect(fspSummary).toContain(`Total payment amount: €${paymentAmount}`);
```

Comment:
Maybe you want to make currency a variable, because this can be different per project

```suggestion
    expect(fspSummary).toContain(`Total payment amount: ${currency}${paymentAmount}`);
```